tests/basic/storage_pool_test.go

```diff
@@ -4585,12 +4585,169 @@ var _ = Describe("{ResizeClusterNoQuorum}", func() {
+		for _, stNode := range stNodes {
+			if len(nodePools) > 1 {
+				nodePools = stNode.StoragePools
+				nodeSelected = stNode
+				break
+			}
+		}
+
+		dash.VerifyFatal(len(nodePools) > 1, true, "Node has multiple storage pools?")
+		var poolToDelete node.StoragePool
+		for _, pl := range nodePools {
+			if pl.ID != 0 {
+				poolToDelete = pl
+				break
+			}
+		}
+		poolIDToDelete := fmt.Sprintf("%d", poolToDelete.ID)
+		poolsMap, err := Inst().V.GetPoolDrives(&nodeSelected)
+		log.FailOnError(err, "error getting pool drive from the node [%s]", nodeSelected.Name)
+		if _, ok := poolsMap[poolIDToDelete]; !ok {
+			log.FailOnError(fmt.Errorf("error idetifying pool drive"), "poolID %s not found in the node %s", poolIDToDelete, nodeSelected.Name)
+		}
+
+		poolsBfr, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+		log.FailOnError(err, "Failed to list storage pools")
+
+		stepLog = fmt.Sprintf("Delete poolID %s on node %s", poolIDToDelete, nodeSelected.Name)
+
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			log.InfoD("Setting pools in maintenance on node %s", nodeSelected.Name)
+			err = Inst().V.EnterPoolMaintenance(nodeSelected)
+			log.FailOnError(err, "failed to set pool maintenance mode on node %s", nodeSelected.Name)
+
+			err = Inst().V.DeletePool(nodeSelected, poolIDToDelete)
+			log.FailOnError(err, "failed to delete poolID %s on node %s", poolIDToDelete, nodeSelected.Name)
+
+			err = Inst().V.ExitPoolMaintenance(nodeSelected)
+			log.FailOnError(err, "failed to exit pool maintenance mode on node %s", nodeSelected.Name)
+
+			err = Inst().V.WaitDriverUpOnNode(nodeSelected, 5*time.Minute)
+			log.FailOnError(err, "volume driver down on node %s", nodeSelected.Name)
+
+			poolsAfr, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+			log.FailOnError(err, "Failed to list storage pools")
+
+			dash.VerifySafely(len(poolsBfr) > len(poolsAfr), true, "verify pools count is updated after pools deletion")
+
+			poolsMap, err = Inst().V.GetPoolDrives(&nodeSelected)
+			log.FailOnError(err, "error getting pool drive from the node [%s]", nodeSelected.Name)
+			_, ok := poolsMap[poolIDToDelete]
+			dash.VerifyFatal(ok, false, "verify drive is deleted from the node")
+
+		})
+
+		newSpecSize := (poolToDelete.TotalSize / units.GiB) / 2
+		///creating a spec to perform add  drive
+		driveSpecs, err := GetCloudDriveDeviceSpecs()
+		log.FailOnError(err, "Error getting cloud drive specs")
+
+		deviceSpec := driveSpecs[0]
+		deviceSpecParams := strings.Split(deviceSpec, ",")
+
+		paramsArr := make([]string, 0)
+		for _, param := range deviceSpecParams {
+			if strings.Contains(param, "size") {
+				paramsArr = append(paramsArr, fmt.Sprintf("size=%d,", newSpecSize))
+			} else {
+				paramsArr = append(paramsArr, param)
+			}
+		}
+		newSpec := strings.Join(paramsArr, ",")
+		stepLog = fmt.Sprintf("Adding cloud drive to node %s with size %s", nodeSelected.Name, newSpec)
+
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
```

Comment:
Same as above

Reply:
log.InfoD, we are using it to push to log file and aetos dashboard